<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptDetialWidget</class>
 <widget class="QWidget" name="ReceiptDetialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>76</width>
    <height>52</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>766</width>
    <height>523</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="466,300">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ReceiptDetialTitleWidget" name="titleWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ReceiptDetialListView" name="receiptListView">
           <property name="styleSheet">
            <string notr="true">QListView{
border:none;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="bottomWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>62</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#bottomWidget{
    background-color:#ffffff;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QWidget" name="groupWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>13</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QWidget" name="buttonWidget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="lastButton">
                    <property name="minimumSize">
                     <size>
                      <width>87</width>
                      <height>37</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton
{
        border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
        border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
border-image: url(:/saleImage/btn-up-disable.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_6" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>1</width>
                      <height>18</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>1</width>
                      <height>18</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border: 1px solid #ededed;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="nextButton">
                    <property name="minimumSize">
                     <size>
                      <width>87</width>
                      <height>37</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
        border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed{
        border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled{
        border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>17</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="pageLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size:12px;
color:#999999;
font-family:&quot;微软雅黑&quot;;
padding-top:6px;</string>
                 </property>
                 <property name="text">
                  <string>1/1页</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>86</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="contrlWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>210</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="printButton">
                 <property name="minimumSize">
                  <size>
                   <width>83</width>
                   <height>42</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
        font-size:16px;
        color:#ffffff;
        border-image: url(:/adjustPrice/blueButton.png);
}

QPushButton:pressed{
        border-image: url(:/adjustPrice/blueButtonPress.png);
}

QPushButton:disabled{
        border-image: url(:/adjustPrice/buttonDisable.png);
}</string>
                 </property>
                 <property name="text">
                  <string>打印</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ReceiptDetialInfoWidget" name="receiptDetialWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReceiptDetialTitleWidget</class>
   <extends>QWidget</extends>
   <header>receiptdetialtitlewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiptDetialListView</class>
   <extends>QListView</extends>
   <header>receiptdetiallistview.h</header>
  </customwidget>
  <customwidget>
   <class>ReceiptDetialInfoWidget</class>
   <extends>QWidget</extends>
   <header>receiptdetialinfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
